ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -82,30 +99,38 @@ public void onAnimationEnd(final Animator animation) {
         }
     };
 
+    //region Initialization
     public PageContentLayout(@NonNull final Context context) {
         this(context, null);
+        initialization(context);
```

Comment:
`initialization()` would have already been called twice at this point.

Reply:
I see now those aren't super they are calls to other constructors That is what confused me.